src/figuresClasses.ts

```diff
@@ -1,19 +1,82 @@
 export interface Figure {
-
+  shape: string;
+  color: string;
+  getArea(): number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape: string = 'triangle';
 
-}
+  color: string;
+
+  a: number;
+
+  b: number;
 
-export class Circle {
+  c: number;
 
+  constructor(color: string, a: number, b: number, c: number) {
+    if (a <= 0 || b <= 0 || c <= 0 || a + b <= c || a + c <= b || b + c <= a) {
+      throw new Error('Invalid triangle sides');
+    }
+    this.color = color;
+    this.a = a;
+    this.b = b;
+    this.c = c;
```

Comment:
![image](https://github.com/mate-academy/ts_figures_classes/assets/54860641/dcdaff8d-a594-49bb-b4cb-ab4cfdbfb7a8)
Fix this in all code.